docs/projects/ad9656_fmc/index.rst

```diff
@@ -0,0 +1,287 @@
+featuring an on-chip sample and hold circuit. It is designed to be low cost, low
+power and compact, making it easy to use.
+
+:adi:`AD9656` operates at a conversion rate of up to 125 MSPS, optimized for
+excellent dynamic performance and low power consumption, which is crucial for
+applications requiring a small package size. It requires a single 1.8 V power
+supply and supports LVPECL, CMOS, and LVDS-compatible sample rate clocks for
+full performance. The ADC does not need external reference or driver components
+for many applications. It also includes features to enhance flexibility and
+reduce system costs, such as a programmable output clock, data alignment
+and digital test pattern generation.
+
+An on-chip phase-locked loop (PLL) allows users to provide a single ADC sampling
+clock. The PLL multiplies the ADC sampling clock to produce the corresponding
+JESD204B data rate clock. The configurable JESD204B output block supports up to
+8.0 Gbps per lane. JESD204B output block supports one, two, and four lane
+configurations. The SPI control offers a wide range of flexible features to meet
+specific system requirements
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`EVAL-AD9656 <EVAL-AD9656>`
+
+Supported devices
+-------------------------------------------------------------------------------
+
+-  :adi:`AD9656`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+.. list-table::
+   :widths: 35 35 30
+   :header-rows: 1
+
+   * - Evaluation board
+     - Carrier
+     - FMC slot
+   * - :adi:`EVAL-AD9656 <EVAL-AD9656>`
+     - :xilinx:`ZCU102`
+     - FMC HPC1
+
+Block design
+-------------------------------------------------------------------------------
+
+ZCU102 block diagram
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: ad9656_fmc_xilinx_block_diagram.svg
+   :width: 800
+   :align: center
+   :alt: AD9656-FMC/ZCU102 block diagram
+
+The Rx links (ADC Path) operate with the following parameters:
+
+-  Rx Deframer parameters: L=4, M=4, S=1, NP=16, N=16
+-  Dual link: No
+-  RX_DEVICE_CLK: 62.5 MHz
+-  REF_CLK: 125MHz
+-  JESD204B Lane Rate: 10Gbps
+-  QPLL0 or CPLL
+
+AD9656 FMC Card block diagram
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: ad9656_fmc_connector_diagram.svg
+   :width: 800
+   :align: center
+   :alt: AD9656-FMC/ZCU102 FMC Card block diagram
+
+Clock scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :adi:`AD9656` ADC's default clock input is from an on-board 125MHz crystal
+oscillator, which goes through a transformer-coupled circuit to minimize jitter.
+The :adi:`AD9656` has an internal clock divider (ratios 1-8) for higher
+frequency clocks.
+
+Configuring the clock source for :adi:`AD9656`:
```

Comment:
Rephrase to emphasize that there are actually 2 ways, because now it looks like it is a 2-step configuration